node-manager/src/node.rs

```diff
@@ -33,20 +40,32 @@ pub(crate) type PeerManager = LdkPeerManager<
 pub struct Node {
     pub uuid: String,
     pub pubkey: PublicKey,
+    pub peer_manager: Arc<PeerManager>,
     pub keys_manager: Arc<KeysManager>,
 }
 
 impl Node {
     pub(crate) fn new(node_index: NodeIndex, mnemonic: Mnemonic) -> Result<Self, MutinyError> {
         info!("initialized a new node: {}", node_index.uuid);
 
-        let keys_manager = create_keys_manager(mnemonic, node_index.child_index);
+        let logger = Arc::new(MutinyLogger::default());
+
+        let keys_manager = Arc::new(create_keys_manager(mnemonic, node_index.child_index));
         let pubkey = pubkey_from_keys_manager(&keys_manager);
 
+        let ln_msg_handler = MessageHandler {
+            chan_handler: Arc::new(ErroringMessageHandler::new()),
```

Comment:
We're going to end up putting the channel manager here, could you just add a comment mentioning that so we don't forget?